composer.json

```diff
@@ -17,12 +17,12 @@
     ],
     "require": {
         "php": "^7.1",
-        "phpstan/phpstan": "^0.11"
+        "phpstan/phpstan": "~0.11"
```

Comment:
Better to keep `^` operator for versions 0.* because of potential BC breaks, which is the case here (see Travis output).

Reply:
^0.11 prevents installation of phpstan 0.12

Reply:
Yep, but you can use `^0.12`

Reply:
well, this change should not break BC with 0.11.* and so I intended to keep it, maybe "^0.11|^0.12" what do you think?